playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -277,20 +292,47 @@ def debugger_stack_trace(event: PodEvent, params: DebuggerParams):
         finding_type=FindingType.REPORT,
         failure=False,
     )
-    cmd = f"debug-toolkit stack-trace {pid}"
+    cmd = f"debug-toolkit stack-trace {pid} --amount={params.traces_amount} --sleep-duration-s={params.sleep_duration_s}"
     output = RobustaPod.exec_in_debugger_pod(
         pod.metadata.name,
         pod.spec.nodeName,
         cmd,
     )
     blocks = []
-    for thread_output in output.split("\n\n"):
-        if thread_output.startswith("Current thread"):
-            # this is the thread we are getting the stack trace from, not relevant for debugging
-            continue
-        if thread_output:
-            blocks.append(MarkdownBlock(f"```\n{thread_output}\n```"))
-    finding.add_enrichment(blocks, annotations={SlackAnnotations.ATTACHMENT: True})
+    try:
+        output_json = json.loads(output)
+        if len(output_json) == 0 or (len(output_json) == 1 and output_json[0]["status"].lower() != "success"):
```

Comment:
What is the data on output_json[0] ?
it has status, error, and trace fields?
A little hard to understand

Reply:
each stack trace object has

- time
- status
- error
- trace
fields, I can throw it in an enum
we only show the time and trace outputs in findings and error is printed to log if there is an error